week03/13. Roman to Integer.cpp

```diff
@@ -0,0 +1,43 @@
+class Solution {
+public:
+    int romanToInt(string s) {
+
+        unordered_map<char, int> roman = {
+            {'I', 1},
+            {'V', 5},
+            {'X', 10},
+            {'L', 50},
+            {'C', 100},
+            {'D', 500},
+            {'M', 1000}
+        };

```

Comment:
Why are you initializing your map inside this function? It will be initiated each time this function gets called. Just put it outside the function as a member variable of this class.

Reply:
I may study about local variable and global variable use purpose and need to clear my concept on them. That's why at first I could not understand how to use. Thanks for your suggestion...